tests/test_profile.py

```diff
@@ -62,49 +37,31 @@ def test_edit_profiles_website(self, mozwebqa, existing_user):
 
         # update profile website to include a timestamp
         profile_page.update_profile_website(new_url)
-        actual_website = profile_page.profile_website
-
-        Assert.equal(actual_website, new_url,
-                     "update to website on profile edit page. \
-                     Expected '%s' but returned '%s'" %
-                     (new_url, actual_website))
+        assert new_url == profile_page.profile_website, 'Website did not update'
 
         # verify username persists after logging out and then logging back in
         logged_out = profile_page.logout()
         home_page = logged_out.login(existing_user['email'], existing_user['password'])
         profile_page = home_page.click_profile()
-
-        actual_website = profile_page.profile_website
-
-        Assert.equal(actual_website, new_url,
-                     "update to website did not persist after logout. \
-                     Expected '%s', but returned '%s'" %
-                     (new_url, actual_website))
+        assert new_url == profile_page.profile_website, 'Website change did not persist after logout'
 
         # verify user can leave website field empty
         profile_page.update_profile_website("")
-        actual_website = profile_page.profile_website
-
-        Assert.equal(actual_website, "",
-                     "user can't set website URL to an empty string. "
-                     "Expected '', returned '%s'" % actual_website)
+        assert '' == profile_page.profile_website, 'Blank website was not accepted'
 
-    @destructive
     def test_verify_layout_logged_in_user(self, mozwebqa, existing_user):
         start_page = StartPage(mozwebqa)
         home_page = start_page.login(existing_user['email'], existing_user['password'])
         edit_page = home_page.click_profile()
+        assert edit_page.is_stats_section_visible()
```

Comment:
I'm not sure we want to address this nit, but I notice that in other cases (e.g., `is_the_current_page` and `is_user_logged_in`) the boolean check functions are properties, but on this page all of these `is_x` checks are not.